lib/misty/openstack/swift/v1.rb

```diff
@@ -18,6 +18,14 @@ def self.prefix_path_to_ignore
         def self.service_names
           %w{object-storage object-store}
         end
+
+        # Custom requests
+
+        def bulk_delete(data)
```

Comment:
I see one problem to define bulk_delete here. Because `requests()` in [client](https://github.com/flystack/misty/blob/master/lib/misty/http/client.rb#L28) will not recognize `bulk_delete` 


In my case I use the `request()` function in a client_wrapper to  define missing methods for requests. So when I use the wrapper ``bulk_delete`` is unkown

Reply:
Do you have a link example?

Reply:
sure! [here](https://github.com/sapcc/elektra/blob/master/lib/core/api/client_wrapper.rb#L90)
the ``service`` is the misty client and is created in the ``client_manager``

Reply:
@hgw77, I got it, thanks. You're right, darn, I should have made requests method private, :D). Seriously, https://github.com/flystack/misty/pull/93 is allowing to add wrapper methods properly.